client/extensions/woocommerce/state/sites/payment-methods/reducer.js

```diff
@@ -4,27 +4,37 @@
 import { createReducer } from 'state/utils';
 import { LOADING } from 'woocommerce/state/constants';
 import {
-	WOOCOMMERCE_PAYMENT_METHODS_REQUEST,
-	WOOCOMMERCE_PAYMENT_METHODS_REQUEST_SUCCESS,
+	WOOCOMMERCE_PAYMENT_METHOD_ENABLED_UPDATE,
+	WOOCOMMERCE_PAYMENT_METHOD_ENABLED_UPDATE_SUCCESS,
 	WOOCOMMERCE_PAYMENT_METHOD_UPDATE,
 	WOOCOMMERCE_PAYMENT_METHOD_UPDATE_SUCCESS,
+	WOOCOMMERCE_PAYMENT_METHODS_REQUEST,
+	WOOCOMMERCE_PAYMENT_METHODS_REQUEST_SUCCESS,
+
 } from 'woocommerce/state/action-types';
 
 // TODO: Handle error
 
 export default createReducer( {}, {
-	[ WOOCOMMERCE_PAYMENT_METHODS_REQUEST ]: () => {
-		return LOADING;
+	[ WOOCOMMERCE_PAYMENT_METHOD_ENABLED_UPDATE ]: ( state ) => {
+		// TODO: Return some sort of saving indicator
```

Comment:
Careful - state is equal to the array of methods here, right? You might want the comment to say "Return some sort of saving indicator on the appropriate method in the passed array" instead